ddp-workspace/projects/toolkit-prion/tsconfig.lib.json

```diff
@@ -0,0 +1,46 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/lib",
+    "target": "es5",
+    "module": "es2015",
+    "moduleResolution": "node",
+    "declaration": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "types": [],
+    "lib": [
+      "dom",
+      "es2018"
+    ],
+    "paths": {
+      "ddp-sdk": [
+        "dist/ddp-sdk"
+      ],
+      "ddp-sdk/*": [
+        "dist/ddp-sdk/*"
+      ],
+      "toolkit": [
+        "dist/toolkit"
+      ],
+      "toolkit/*": [
+        "dist/toolkit/*"
+      ]
+    }
+  },
```

Comment:
Are you getting errors in IDE when you do an `import { Something} from 'ddp-sdk'` or from 'toolkit'?
I do. My problem is resolved when I remove the the whole "paths": entry from this file.
"paths" gets inherited from main workspace tsconfig.json so probably not needed here.
Should point out that this setting does not appear to make a difference when doing `ng build` or `ng serve` one way or the other.